docs/questions/sharing/visualizations/pie-or-donut-chart.md

```diff
@@ -1,11 +1,101 @@
+
+You can rename, reorder, or remove slices from the inner ring of a sunburst or pie chart. To reorder the pie slices, drag the cards with the slice names. To rename the slices, click on three dots next to the series name and enter a new name.
+
+To change the color of the slices, click on the color circle next to the slice name. In sunburst charts, you can only change the color of the inner ring slices (the outer ring slices inherit the color of their parent inner slice).
+
+If your query has multiple numeric metrics (columns), you can pick the column that should be depicted on the chart in the **Measure** dropdown.
+
+To format the total displayed in the middle of the chart, click on the three dots next to the metric name in the **Measure** setting. The measure format options — including the "number of decimal places" option — will only apply to the total, and not to the percentage values or labels. To configure the display of the percentage values and labels, go to the [display settings](#display-settings) tab.
+
+![format sunburst](../../images/sunburst-metric-options.png)
+
+### Display settings
+
+You can configure whether to show:
+
+- The legend next to the chart.
+- The total in the center of the chart. The format of the total can be changed in the [data settings](#data-settings).
+- The labels for the slices. The labels for the inner ring slices can be changed in the [data settings](#data-settings).
+
+  For sunburst charts with multiple breakouts, only the inner ring breakout will be displayed in the legend, and by default, "Show labels" will be turned on, so the labels for slices will be visible on the chart. If you turn off "Show labels" for a sunburst chart, the only way to distinguish slices will be to hover over them.
+
+- The percentage values for the slices. If you select to display percentages **In the legend**, but you toggle off **Show legend**, the percentage values will not be shown on the chart.
+
+  You can always see percentage values for any slice by hovering over the slice.
+
+To change the number of decimal places in the percentage values, use the **Number of decimal places** setting in the **Display** tab. If you want to change the number of decimal places in display of the total in the center of the chart, go to [data settings](#data-settings).
+
+![Settings showcase](../../images/sunburst-settings-showcase.png)
+
+To make the chart more legible, you can group slices smaller than a certain percentage into one slice by adjusting **Minimum slice percentage**. You'll be able to see the categories and values in the **Other** slice by hovering over it:
+
+![Other slice](../../images/pie-other-slice.png)
+
+Currently, you can't change the color or label of the **Other** slice.
+
+## Limitations and alternatives
+
+Consider using a bar chart (or a [stacked bar chart](line-bar-and-area-charts.md#stacked-bar-chart)) or a [pivot table](pivot-table.md) instead of a pie or sunburst chart in the following cases:
```

Comment:
```suggestion
Consider using a bar or row chart (or a [stacked bar chart](line-bar-and-area-charts.md#stacked-bar-chart)) or a [pivot table](pivot-table.md) instead of a pie or sunburst chart in the following cases:
```